test/functional/collection_management_spec.test.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const { expect } = require('chai');
+const { loadSpecTests } = require('../spec/index');
+const { runUnifiedTest } = require('./unified-spec-runner/runner');
+
+describe('Collection management unified spec tests', function () {
+  for (const collectionManagementTest of loadSpecTests('collection-management')) {
+    expect(collectionManagementTest).to.exist;
```

Comment:
just a suggestion: I generally recommend against using assertions that aren't function calls (like `exist`) because they're easy to accidentally disable with a typo (e.g., `.exists` will resolve to `undefined` so the "assertion" will continue to pass even though it's no longer testing anything); it's not a major problem since the IDE (and possibly the linter) will yell at you if you do try to access an undefined assertion, but something like `.to.be.an('object')` would technically be safer

Reply:
Running a test with a typo like that would result in a failure with an error like `Error: Invalid Chai property: exits. Did you mean "exists"?`.

Reply:
Oh that's awesome, they must have updated it in one of the recent-ish versions, because my favorite example used to be `expect(1).to.be.a.potato`. I fully withdraw my objection then :)

Reply:
TIL 🚀